docs/source/docker/integration/s3fs-fuse.rst

```diff
@@ -0,0 +1,213 @@
+        are supported and installed on your system.
+    perl: warning: Falling back to a fallback locale ("C.UTF-8").
+    Selecting previously unselected package s3fs.
+    (Reading database ... 109361 files and directories currently installed.)
+    Preparing to unpack .../archives/s3fs_1.84-1_amd64.deb ...
+    Unpacking s3fs (1.84-1) ...
+    Setting up s3fs (1.84-1) ...
+    Processing triggers for man-db (2.8.5-2) ...
+
+
+Configure S3 credentials
+--------------------------
+
+Our freshly installed `s3fs-fuse <https://github.com/s3fs-fuse/s3fs-fuse>`_ 
+requires access to our S3 bucket. Luckily we can
+directly use a AWS credentials file. This file should look like this:
+
+.. code-block:: yaml
+    :caption: Example ./aws/credentials
+
+    [default]
+    aws_access_key_id=AKIAIOSFODNN7EXAMPLE
+    aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+Let's mount a bucket. We need to create a local folder where we want the S3
+content to be mounted on. 
+
+.. code-block:: console
+    :caption: Create a local folder **/s3-mount** using the terminal
+
+    mkdir /s3-mount
+
+
+Now let's use s3fs to mount the bucket to our new folder. Run the following command
+in your terminal.
+
+.. code-block:: console
+  :caption: Using the default aws credentials from **~/.aws/credentials**
+
+  s3fs simple-test-bucket-igor /s3-mount
+
+.. note:: If you don't specify the location of the `.passwd_file` s3fs uses the default
+          location of your aws credentials **~/.aws/credentials**.
+
+
+If everything went well you should now be able to see the content of your bucket
+in your **/s3-mount** folder. If you add a new file to the folder it will 
+automatically be uploaded to the bucket.
+
+Optional: use a custom .passwd file for s3fs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't want to use the default aws credentials you can also create a separate
+passwd file for s3fs:
+
+.. code-block:: console
+
+    echo ACCESS_KEY_ID:SECRET_ACCESS_KEY > ${HOME}/.passwd-s3fs
+    chmod 600 ${HOME}/.passwd-s3fs
+
+
+Now we can mount the S3 bucket using the following command in the terminal.
+
+.. code-block:: console
+  :caption: Using the credentials in the **.passwd-s3fs** file
+
+  s3fs simple-test-bucket-igor /s3-mount -o passwd_file=${HOME}/.passwd-s3fs
+
+Use the S3 storage for Lightly Docker
+---------------------------------------
+
+Now we can use the docker run command and use the `/s3-mount` directory as the
+input dir.
+
+.. code-block:: console
+
+    docker run --gpus all --rm -it \
+        -v /s3-mount:/home/input_dir:ro \
+        -v /docker/output:/home/output_dir \
+        lightly/sampling:latest \
+        token=MYAWESOMETOKEN
+
+You can do the same for the docker output directory (in this example I used 
+`/docker/output`). We can either use the same bucket and work on subfolders
+or use another bucket and repeat the procedure.
+
+If you use a mounted S3 bucket for the docker output can be very handy.
```

Comment:
I think it should be "Using a mounted S3 bucket for the docker output can be very handy."